Add option: useSrc

AlmogBaku opened the pull request:
Add an option to use the image's src instead of canvas.toDataUrl() in the parent image-background.

this commit add boolean setting named "useSrc"

Commit:
Added test file and test directory.

jasonwyatt commented:
Thanks for the pull request, I tested your changes against my test file and saw that it's not behaving properly when the image element has received CSS styling.  This screenshot is from the test file, but with your changes and `useSrc: true`:

![Imgur](http://i.imgur.com/a8lpa.png)

I've pushed the test file to master so you can use it in your test as well.

AlmogBaku committed:
sourceImage fixing:
  - fixing padding issue
  - change the setting name
  - fixing for float right issues

AlmogBaku commented:
Hey Jason!
I found the problem.. the problem was the padding.. I re-pulled it now :) (with tests!)

AlmogBaku committed:
fixing canvas.toDataURL()

jasonwyatt commented:
Thanks @AlmogBaku !
